website/phpinfo.php

```diff
@@ -0,0 +1,3 @@
+<?php
```

Comment:
Should this really land on production instance?

Reply:
clean miss, good catch, removed